tuxemon/config.py

```diff
@@ -200,6 +202,8 @@ def get_defaults() -> Mapping[str, Any]:
                         ("locale", "en_US"),
                         ("dev_tools", False),
                         ("recompile_translations", True),
+                        ("compressed_save", False),
+                        ("compressed_save_use_lzma", False),
```

Comment:
I would have a single key with possible values `None` (the default) and a string with the name of a compression module "lzma", "zipfile", whatever.
```suggestion
                        ("compression", None),
```